x-pack/plugins/cases/public/containers/use_get_case_user_actions.test.tsx

```diff
@@ -21,97 +20,52 @@ import {
-        useGetCaseUserActions(basicCase.id, basicCase.connector.id)
-      );
-      await waitForNextUpdate();
-      expect(result.current).toEqual({
-        ...initialData,
-        fetchCaseUserActions: result.current.fetchCaseUserActions,
-      });
-    });
-  });
-
-  it('calls getCaseUserActions with correct arguments', async () => {
-    const spyOnPostCase = jest.spyOn(api, 'getCaseUserActions');
-
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
-        useGetCaseUserActions(basicCase.id, basicCase.connector.id)
-      );
-      await waitForNextUpdate();
-
-      result.current.fetchCaseUserActions(basicCase.id, basicCase.connector.id);
-      await waitForNextUpdate();
-      expect(spyOnPostCase).toBeCalledWith(basicCase.id, abortCtrl.signal);
-    });
-  });
-
   it('returns proper state on getCaseUserActions', async () => {
     await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
-        useGetCaseUserActions(basicCase.id, basicCase.connector.id)
-      );
-      await waitForNextUpdate();
-      result.current.fetchCaseUserActions(basicCase.id, basicCase.connector.id);
-      await waitForNextUpdate();
-      expect(result.current).toEqual({
-        ...initialData,
-        caseUserActions,
-        fetchCaseUserActions: result.current.fetchCaseUserActions,
-        hasDataToPush: true,
-        isError: false,
-        isLoading: false,
-        participants: [elasticUser],
-      });
-    });
-  });
-
-  it('set isLoading to true when posting case', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
-        useGetCaseUserActions(basicCase.id, basicCase.connector.id)
-      );
-      await waitForNextUpdate();
-      result.current.fetchCaseUserActions(basicCase.id, basicCase.connector.id);
-
-      expect(result.current.isLoading).toBe(true);
-    });
-  });
-
-  it('unhappy path', async () => {
-    const spyOnPostCase = jest.spyOn(api, 'getCaseUserActions');
-    spyOnPostCase.mockImplementation(() => {
-      throw new Error('Something went wrong');
-    });
-
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseGetCaseUserActions>(() =>
-        useGetCaseUserActions(basicCase.id, basicCase.connector.id)
+      const { result } = renderHook<string, UseGetCaseUserActions>(
+        () => useGetCaseUserActions(basicCase.id, basicCase.connector.id),
+        { wrapper }
       );
-      await waitForNextUpdate();
-      result.current.fetchCaseUserActions(basicCase.id, basicCase.connector.id);
-
-      expect(result.current).toEqual({
-        ...initialData,
-        isLoading: false,
-        isError: true,
-        fetchCaseUserActions: result.current.fetchCaseUserActions,
+      await waitFor(() => {
```

Comment:
nit: test for the error toaster is missing.

Reply:
I added the test